bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -49,15 +49,8 @@ apiServerArguments:
     - /etc/kubernetes/secrets/etcd-client.key
   etcd-servers: {{range .EtcdServerURLs}}
     - {{.}}{{end}}
-  feature-gates:
-    - "APIPriorityAndFairness=true"
-    - "RotateKubeletServerCertificate=true"
-    - "DownwardAPIHugePages=true"
-    - "CSIMigrationAzureFile=false"
-    - "CSIMigrationvSphere=false"
-{{- if .ServiceCIDR | len | eq 2}}
-    - "IPv6DualStack=true"
-{{- end}}
```

Comment:
Is there a place, where I could verify that the following flags are handed down as `.FeaturesGates`?

Reply:
Helpful: https://github.com/openshift/cluster-kube-apiserver-operator/pull/1419/files#r1064687393